generation/localizedstringkit/detection.py

```diff
@@ -113,10 +113,20 @@ def _detect_strings(self, patterns: List[Tuple[Pattern, int]]) -> List[Localized
         # Find occurrences of `Localized` function calls
         matches_in_buffer: List[Tuple] = []
         for pattern, count in patterns:
-            matches_in_buffer += self._get_matches(pattern, count)
+            # Append source Pattern to tuple to discern if bundle is overridden
+            match = self._get_matches(pattern, count) + (pattern)
+            matches_in_buffer += match
 
         results = []
 
+        # Overriden bundle patterns to check
+        SWIFT_LOCALIZED_BUNDLE_PATTERN: Pattern = re.compile(
```

Comment:
The detector shouldn't know about any of its subclasses. It should be left generic.

Reply:
(You can always pass in parameters though)./

Reply:
Updated to take a `bundle_pattern` param and subclasses pass it in.